app/controllers/organizations/pets_controller.rb

```diff
@@ -1,34 +1,35 @@
 class Organizations::PetsController < Organizations::BaseController
+  verify_authorized
```

Comment:
So, for my benefit (have not looked at the docs so excuse my laziness) this method will run the prechecks in the org pet policy based on naming convention? i.e. Organizations::PetPolicy

Reply:
This method is actually used to check that every action in the controller has a policy check called within it. For example, if `#new` did not have any authorization logic in it, an error would raise when calling that action. The final form should have this call only in `ApplicationController`. That way, all controller actions should require some form of authorization. I am currently adding it individually to controllers as I work on them so that I can still navigate to unauthorized pages when testing stuff in development.

There is also `skip_verify_authorized` that can be used to skip this requirement for specific actions, like seen here:
https://github.com/mononoken/pet-rescue/blob/e1d6102e4b7877242fc1cc532132bbcbc57114a7/app/policies/adoptable_pet_policy.rb#L2C3-L2C57

Reply:
Ah very cool. Ok thank you for the explanation!